Description: Install the static library again
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed
Last-Update: 2024-09-02

--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -432,6 +432,13 @@ if (BUILD_STATIC_LIB)
     if (WITH_COVERAGE)
         append_coverage_compiler_flags_to_target(ssh-static)
     endif (WITH_COVERAGE)
+
+  install(TARGETS
+            ssh-static
+          DESTINATION
+            ${LIB_INSTALL_DIR}/${OUTPUT_SUFFIX}
+          COMPONENT
+            libraries)
 endif (BUILD_STATIC_LIB)
 
 message(STATUS "Threads_FOUND=${Threads_FOUND}")
